src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/managehearings/ManageHearingActionService.java

```diff
@@ -161,7 +162,10 @@ private void updateTabData(FinremCaseData caseData) {
                 .build())
             .toList();
 
-        caseData.getManageHearingsWrapper().setHearingTabItems(hearingTabItems);
+        List<HearingTabCollectionItem> existingHearingTabItems =
+            new ArrayList<>(emptyIfNull(caseData.getManageHearingsWrapper().getHearingTabItems()));
+        existingHearingTabItems.addAll(hearingTabItems);
+        caseData.getManageHearingsWrapper().setHearingTabItems(existingHearingTabItems);
```

Comment:
So the purpose of this is to populate hearing tabs for hearings.

So I can see why this would overwrite the tab data introduced by migration.

However, it looks like this will keep appending getHearingTabItems data to itself each time it's run when there are actual hearings.  Can you check?

Reply:
oh... You're right. I have to think about how to fix it.